napari/layers/_shapes_layer/shape_util.py

```diff
@@ -678,3 +678,91 @@ def segment_normal(a, b):
     else:
         unit_norm = normal/norm
     return unit_norm
+
+
+def poly_to_mask(mask_shape, vertices):
+    """Converts a polygon to a boolean mask with `True` for points
+    lying inside the shape. Uses the bounding box of the vertices to reduce
+    computation time.
+
+    Parameters
+    ----------
+    mask_shape : np.ndarray | tuple
+        1x2 array of shape of mask to be generated.
+    vertices : np.ndarray
+        Nx2 array of the vertices of the polygon.
+
+    Returns
+    ----------
+    mask : np.ndarray
+        Boolean array with `True` for points inside the polygon
+    """
+    mask = np.zeros(mask_shape, dtype=bool)
+    bottom = vertices.min(axis=0).astype('int')
```

Comment:
it's recommended to use NumPy types:
```suggestion
    bottom = vertices.min(axis=0).astype(np.int)
```

Reply:
fixed